docs/gettingstarted/index.rst

```diff
@@ -1,266 +0,0 @@
-============================
-
-.. logo::
-
-.. intro-text::
-
-   Flocker lets you run microservices apps with database containers and move them around between servers.
-   It comes in two pieces and you'll need both.
-
-.. contents::
-   :local:
-
-.. tutorial-step::
-
-   Step 1: Installing Flocker Client & Node
-   ========================================
-
-   .. parallel::
-
-      .. mobile-label::
-
-         Local
-
-      .. image:: images/macbook.png
-         :class: center-block img-responsive
-         :alt: Flocker Client diagram
-
-      Flocker Client
-      --------------
-
-      +--------------------------------------------------------------------------+
-      | Runs on your laptop                                                      |
-      +--------------------------------------------------------------------------+
-      | Uses application and deployment configuration files                      |
-      +--------------------------------------------------------------------------+
-      | Deploys containers to a cluster of servers                               |
-      +--------------------------------------------------------------------------+
-
-      .. empty-div:: arrow-down center-block hidden-xs hidden-sm
-
-   .. parallel::
-
-      .. mobile-label::
-
-         Live
-
-      .. image:: images/nodes.png
-         :class: center-block offset-top img-responsive
-         :alt: Flocker Node diagram
-
-      Flocker Node
-      ------------
-
-      +--------------------------------------------------------------------------+
-      | Runs on each server in a cluster                                         |
-      +--------------------------------------------------------------------------+
-      | Links, ports and volumes work across hosts                               |
-      +--------------------------------------------------------------------------+
-      | After deployment, containers can move around                             |
-      +--------------------------------------------------------------------------+
-
-      .. empty-div:: arrow-down center-block
-
-   .. parallel::
-
-      Installation
-      ------------
-
-      .. mobile-label::
-
-         Local
-
-      .. tabs::
-
-         OS X
-         ^^^^
-
-         Install the Flocker client on your Mac (requires Homebrew):
-
-         .. task:: test_homebrew flocker-|latest-installable|
-            :prompt: you@laptop:~$
-
-         Ubuntu 14.04
-         ^^^^^^^^^^^^
-
-         Install the Flocker client on your Linux machine:
-
-         .. task:: install_cli ubuntu-14.04
-            :prompt: you@laptop:~$
-
-         Ubuntu 15.04
-         ^^^^^^^^^^^^
-
-         Install the Flocker client on your Linux machine:
-
-         .. task:: install_cli ubuntu-15.04
-            :prompt: you@laptop:~$
-
-      .. noscript-content::
-
-         OS X
-         ^^^^
-
-         Install the Flocker client on your Mac (requires Homebrew):
```

Comment:
Can you add include 'Xcode' into this line?
https://github.com/ClusterHQ/flocker/pull/1589